taiga_halo2/benches/vp_proof.rs

```diff
@@ -6,7 +6,7 @@ use pasta_curves::pallas;
 use rand::rngs::OsRng;
 use rand::Rng;
 use taiga_halo2::{
-    circuit::{vp_circuit::ValidityPredicateInfo, vp_examples::TrivialValidityPredicateCircuit},
+    circuit::{vp_circuit::ValidityPredicateCircuit, vp_examples::TrivialValidityPredicateCircuit},
```

Comment:
Probably should have changed with the commit that removed it, but good that the bench still wroks.